polymetis/polymetis/environment.yml

```diff
@@ -34,7 +34,7 @@ dependencies:
     - breathe
     - dash
     - grpcio ==1.46.0
-    - habitat-sim=0.2.1=py3.8_bullet_linux_fc7fb11ccec407753a73ab810d1dbb5f57d0f9b9
+    - habitat-sim=0.2.2=py3.8_bullet_linux_011191f65f37587f5a5452a93d840b5684593a00
```

Comment:
Polymetis provides two installation methods: 1) Building locally into an conda env created by this `environment.yml`, or 2) Installing from our prebuilt conda package.

Thus, whenever you update a dependency in the `environment.yml` file, you also need to update the corresponding dependency in our [conda package metadata](https://github.com/facebookresearch/fairo/blob/8cd347337d3f21258a653be0e90d0e369f49def5/polymetis/polymetis/conda/conda_recipe/meta.yaml) to ensure that the two installation methods are aligned.

p.s. Sorry, I know it's not ideal having duplicated dependency specifications in the codebase, but until we implement a better solution this is how we have to do things.